webapp/src/ts/services/rules-engine.service.ts

```diff
@@ -310,73 +347,98 @@ export class RulesEngineService implements OnDestroy {
   }
 
   isEnabled() {
-    return this.initialized.then(this.rulesEngineCore.isEnabled);
+    return this.initialized;
+  }
+
+  private refreshEmissions() {
+    return this.ngZone.runOutsideAngular(() => this._refreshEmissions());
+  }
+
+  private async _refreshEmissions() {
+    const trackName = [ 'rules-engine', 'emissions', 'all-contacts' ];
```

Comment:
I think we should keep emissions as a word fore core devs whenever possible.  Did you consider a name like `rules-engine:background:all-contacts` or `rules-engine:proactive:all-contacts` or alike? Might be more intuitive. Nobody wants to know what emissions are.